### DISCUSSION TITLE
I can't Get Access to the GitHub Copilot.

### DISCUSSION THREAD

**Discussion by omprakashthakur** [community/community#111320] [GitHub Education] [answered]
Subject: Need Assistance with GitHub Copilot Access in Student Program

Hey everyone,

I hope this message finds you well. I'm Om Prakash Thakur, and I recently enrolled in the GitHub Students Program. I'm excited about the opportunities it offers, but unfortunately, I'm facing issues accessing GitHub Copilot.

If anyone has experienced a similar situation or has insights on how to resolve this, I would greatly appreciate your guidance. Your help could make a significant difference in my learning journey.

Thanks in advance for your assistance!

Best regards,
Om Prakash Thakur

**Comment by SebastienXia** [comment]
### Hi, @omprakashthakur and welcome to the **_GITHUB COMMUNITY_** ❤️

> Have you tried with this [link](https://github.com/features/copilot), with your github account?

**Reply by DNXL-UMK** [reply, marked as answer]
just open this link and sign up for it. Have same issues also and solved, cheers. github.com/github-copilot/free_signup

**Reply by omprakashthakur** [reply]
problem is solved. Thank you
